analyses/doublet-detection/scripts/01a_run-scdblfinder.R

```diff
@@ -114,20 +114,11 @@ opts <- parse_args(OptionParser(option_list = option_list))
 if (!file.exists(opts$input_sce_file)) {
     glue::glue("Could not find input SCE file, expected at: `{opts$input_sce_file}`.")
 }
-if (is.null(opts$results_dir)) {
-  stop("Must provide an output path for results with --results_dir.")
+if (!stringr::str_ends(opts$output_tsv_file, ".tsv")) {
```

Comment:
You didn't set a default for this arg, so `NULL` is still possible which will break this. I might give the arg a default value of "scdblfinder_results.tsv" and leave this as is. (I have come to dislike NULL defaults for reasons like this... I get tired of having to include `is.null()` checks everywhere)